core/src/main/resources/hudson/ProxyConfiguration/config.groovy

```diff
@@ -11,8 +11,8 @@ f.entry(title:_("Port"),field:"port") {
 f.entry(title:_("User name"),field:"userName") {
     f.textbox()
 }
-f.entry(title:_("Password"),field:"password") {
-    f.password(value:instance?.encryptedPassword)
+f.entry(title:_("Password"),field:"secretPassword") {
```

Comment:
Field renamed without adapting `validateButton` arguments in https://github.com/jenkinsci/jenkins/pull/3935/files#diff-4af47ca230111a1e83ea06084040271fR25

Fix attempt in https://github.com/jenkinsci/jenkins/pull/4607/commits/8dcf818f654d391646b75fa700b77b9e5c623086